k8s/bigchaindb/bigchaindb-dep.yaml

```diff
@@ -59,8 +41,15 @@ spec:
           value: 3CjmRhu718gT1Wkba3LfdqX5pfYuBdaMPLd7ENUga5dm
         - name: BIGCHAINDB_BACKLOG_REASSIGN_DELAY
           value: "120"
-        - name: BIGCHAINDB_KEYRING
-          value: ""
+        - name: BIGCHAINDB_DATABASE_MAXTRIES
+          value: "3"
+        - name: BIGCHAINDB_DATABASE_CONNECTION_TIMEOUT
+          value: "120"
+        - name: BIGCHAINDB_LOG_LEVEL_CONSOLE
+          value: debug
+        # The following env var is not required for the bootstrap/first node
+        #- name: BIGCHAINDB_KEYRING
+        #  value: ""
```

Comment:
We can handle this in a separate PR about configuring BigchainDB. We will also have to move to Secrets in a separate PR.
Since the list of config options are huge now, we will trim these too, esp for the first node setup.